src/main/java/org/commcare/formplayer/postgresutil/PostgresDB.java

```diff
@@ -0,0 +1,84 @@
+package org.commcare.formplayer.postgresutil;
+
+import org.commcare.formplayer.exceptions.SQLiteRuntimeException;
+import org.commcare.formplayer.services.ConnectionHandler;
+import org.commcare.formplayer.sqlitedb.DBPath;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Properties;
+
+/**
+ * @author $|-|!˅@M
+ */
+public class PostgresDB implements ConnectionHandler {
+    private DBPath dbPath;
+    private Connection connection;
+
+    public PostgresDB(DBPath dbPath) {
+        this.dbPath = dbPath;
+        if (dbPath != null) {
+            // Create the schema before doing anything.
+            Connection connection = getConnection();
+            String schemaCreationSt = "CREATE SCHEMA IF NOT EXISTS " + getCurrentSchema() + ";";
+            PreparedStatement preparedStatement = null;
+            try {
+                preparedStatement = connection.prepareStatement(schemaCreationSt);
+                preparedStatement.execute();
+                preparedStatement.close();
+            } catch (SQLException e) {
+                throw new SQLiteRuntimeException(e);
+            } finally {
+                if (preparedStatement != null) {
+                    try {
+                        preparedStatement.close();
+                    } catch (SQLException e) {
+                        e.printStackTrace();
+                    }
+                }
+            }
+        }
+    }
+
+    public String getCurrentSchema() {
+        return dbPath.getDatabasePath();
+    }
+
+    private Connection getNewConnection() throws SQLException {
```

Comment:
I don't know how spring manages DB connections but maybe we can look at it at a later step. 
We're planning to have a wrapper so that [we can use both sqlite and postgres connection and compare the two](https://dimagi-dev.atlassian.net/browse/SAAS-11692). I think once that's done then we can perhaps see how to utilize spring in managing those connection.